### PR TITLE
Fix signin redirect dev

### LINKED ISSUE #119: As a developer, I need to go to dashboard after login

### PR DESCRIPTION
**Pull request by nuriyealp**
## Description

Fix sign-in redirection to dashboard
## Related Issue
closes #119 

## Acceptance Criteria
Go to the dashboard after login

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     ✓| :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

Before logging in

- Try to go to private routes like `/dashboard`, `/profile` etc.
- Make sure they are not accessible and redirected to landing page

After logging in

- After clicking sign in, observe user is redirected to `/dashboard`
- Observe correct user name is displayed

### CODE REVIEW

**Review by XeniyaDob — APPROVED** [review]
Good job!

File: src/App.js [hunk 1]
```diff
@@ -45,21 +40,33 @@ function App() {
...
+              element={<Filter />}
+            />
+            <Route
+              path="searchresult/:search/:intolerances"
+              element={<SearchResult />}
+            />
+            <Route
+              path="searchresult/:search/:intolerances/:type"
+              element={<Filter />}
+            />
```
Comment (elenamagay) [inline src/App.js:65]: why routes are duplicated?

**Review by elenamagay — CHANGES_REQUESTED** [review]
It works now but you have several duplicated routes.

File: src/App.js (hunk 1 above)
Comment (nuriyealp) [reply src/App.js]: I fixed it.Thank you!

**Review by elenamagay — APPROVED** [review]
Ok, looks good to me now!
You have some merge conflicts. Please resolve before merging the branch

**Review by elenamagay — APPROVED** [review]

**Pull request merged**